spec/decorators/assignment_repo_decorator_spec.rb

```diff
@@ -0,0 +1,110 @@
+require 'rails_helper'
+
+RSpec.describe AssignmentRepoDecorator do
+  let(:user) { User.new }
+  let(:assignment_repo) do
+    AssignmentRepo.new.tap do |repo|
+      repo.user = user
+    end
+  end
+
+  subject { assignment_repo.decorate }
+
+  describe '#avatar_url' do
+    before do
+      allow(user).to receive(:uid).and_return('an_uid')
+    end
+
+    it 'fetches the avatar url in given size' do
+      expect(subject.avatar_url(100))
+        .to eq('https://avatars.githubusercontent.com/u/an_uid?v=3&size=100')
+    end
```

Comment:
ok, can change this, my sublime rubocop plugin defaults to the more restrictive 80 chars per line ;)